jest.config.ts

```diff
@@ -197,11 +197,11 @@ const config: Config = {
 
 	// A map from regular expressions to paths to transformers
 	transform: {
-		'^.+\\.[t|j]sx?$': ['babel-jest', { configFile: './babel.config.jest.js' }],
+		'^.+\\.[t|j]sx?$': 'babel-jest',
```

Comment:
We can unify lines 200 and 204 with this one (204 can be removed then)
```suggestion
		'^.+\\.(m?[t|j]sx?)$': 'babel-jest',
```